charts/fybrik/files/taxonomy/base_taxonomy.json

```diff
@@ -0,0 +1,190 @@
+        "write",
+        "delete",
+        "copy"
+      ]
+    },
+    "DataFormat": {
+      "description": "Format of the data",
+      "type": "string"
+    },
+    "InfrastructureElement": {
+      "type": "object",
+      "required": [
+        "attribute",
+        "value",
+        "object"
+      ],
+      "properties": {
+        "arguments": {
+          "description": "A list of arguments defining a specific metric, e.g. regions for a bandwidth",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "attribute": {
+          "description": "Attribute name defined in the taxonomy",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description",
+          "type": "string"
+        },
+        "instance": {
+          "description": "A reference to the resource instance, e.g. storage account name",
+          "type": "string"
+        },
+        "metricName": {
+          "description": "Name of the metric specified in the metrics section",
+          "type": "string"
+        },
+        "object": {
+          "$ref": "#/definitions/InstanceType",
+          "description": "A resource defined by the attribute (\"fybrikstorageaccount\",\"fybrikmodule\",\"cluster\")"
+        },
+        "value": {
+          "description": "Attribute value",
+          "type": "string"
+        }
+      }
+    },
+    "InfrastructureMetrics": {
+      "type": "object",
+      "required": [
+        "name",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "scale": {
+          "$ref": "#/definitions/RangeType",
+          "description": "A scale of values (minimum and maximum) when applicable"
+        },
+        "type": {
+          "$ref": "#/definitions/AttributeType",
+          "description": "Attribute type, e.g. numeric or string"
+        },
+        "units": {
+          "$ref": "#/definitions/Units",
+          "description": "Measurement units"
+        }
+      }
+    },
+    "InstanceType": {
+      "type": "string",
+      "enum": [
+        "fybrikmodule",
+        "fybrikstorageaccount",
+        "cluster",
+        "inter-region"
+      ]
+    },
+    "Interface": {
+      "type": "object",
+      "required": [
+        "protocol"
+      ],
+      "properties": {
+        "dataformat": {
+          "$ref": "#/definitions/DataFormat",
+          "description": "DataFormat defines the data format type"
+        },
+        "protocol": {
+          "$ref": "#/definitions/ConnectionType",
+          "description": "Protocol defines the interface protocol used for data transactions"
+        }
+      }
+    },
+    "PluginType": {
+      "type": "string"
+    },
+    "PolicyManagerRequestContext": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "ProcessingLocation": {
+      "description": "location information",
+      "type": "string"
```

Comment:
Should the processing location be taken from some taxonomy rather than just being a string?  Should the list of ISO codes for countries (or some other location standard) be in the base taxonomy?  It should definitely be possible to augment or change though, so maybe in a high layer?

Reply:
I think https://github.com/fybrik/fybrik/issues/1052 refers to this issue